images/github-runner/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM ubuntu:latest
+
+
+# system update
+RUN set -eux ; \
+    apt update ; \
+    apt -y upgrade ; \
+    apt -y install \
+            curl \
+            libicu-dev  \
+            build-essential \
+            git \
+            cmake \
+            python3-pip \
+            wget \
+            language-pack-en \
+            libcurl4-openssl-dev;
+
+# cuda install
+# https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html
+ENV DEBIAN_FRONTEND=noninteractive
+RUN set -eux ; \
+    wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/cuda-keyring_1.1-1_all.deb ; \
+    dpkg -i cuda-keyring_1.1-1_all.deb ; \
+    apt-get update ; \
+    apt-get -y install  \
+      cuda-drivers \
+      cuda-nvcc-12-2 \
+      libcublas-dev-12-2;
+
+ARG RUNNER_VERSION=2.314.1
+ARG RUNNER_VERSION_HASH=6c726a118bbe02cd32e222f890e1e476567bf299353a96886ba75b423c1137b5
+
+RUN set -eux ; \
+    mkdir /ggml-ci /tmp/github-runner ; \
+    chown 1000:1000 /ggml-ci /tmp/github-runner ;
+
+WORKDIR /ggml-ci
+
+USER 1000:1000
```

Comment:
This user is not properly created, some scripts may fail if they try to search for username associated to this UID/GID.

Here's the command to create user `1000:1000` used by official [nodejs image](https://hub.docker.com/layers/library/node/lts-bullseye-slim/images/sha256-69d28fb6e9c7e8c2530dc0c2a209d7fcf2fd7413cc16ef023f0eb66ee5bdb20c?context=explore) : 

```
groupadd --gid 1000 node && useradd --uid 1000 --gid node --shell /bin/bash --create-home node
```